src/pudl/metadata/resources/glue.py

```diff
@@ -2,39 +2,20 @@
 from typing import Any
 
 RESOURCE_METADATA: dict[str, dict[str, Any]] = {
-    "assn_gen_eia_unit_epa": {
+    "epacamd_eia": {
         "schema": {
             "fields": [
-                "generator_id",
-                "plant_id_eia",
-                "unit_id_epa",
-            ],
-        },
-        "field_namespace": "glue",
-        "etl_group": "glue",
-        "sources": ["eia_epa_crosswalk"],
-    },
-    "assn_plant_id_eia_epa": {
-        "schema": {
-            "fields": [
-                "plant_id_eia",
                 "plant_id_epa",
+                "emissions_unit_id_epa",
+                "generator_id_epa",
+                "plant_id_eia",
+                "boiler_id",
+                "generator_id",
             ],
```

Comment:
Right now the crosswalk table does not have any primary keys, should it?

Reply:
What set of non-null fields can be used to uniquely identify a row in the table?

Reply:
@zaneselvans

Reply:
It's a little bit convoluted. You need everything, except technically only one of the `plant_ids`. But separating this into two tables seems too convoluted as `plant_id_eia` maps to `plant_id_eia, emission_unit_id_epa, generator_id, boiler_id`.....

Reply:
It's pretty normal for association tables / crosswalks to have all of their columns be part of the primary key, so if you want to set all of them as part of the PK that wouldn't be crazy. So long as none of them are NA -- some databases (including SQLite) won't allow NA values in the PK.

Or if one of the Plant ID is more "real" than the other and you want to pick that one as part of the PK, but not the other plant ID (and it still uniquely identifies all the rows in the table) that would be fine too.

Reply:
Ok, sadly, there are lots of NA values in the `boiler_id` field.

Reply:
Then an auto incrementing pseudo key 🗝️ it is!

Reply:
Do you have and recommendations for what I should call this field? Is there already a standard pseudo key name so I don't have to add something to the metadata?

Reply:
word